tsconfig.json

```diff
@@ -18,6 +18,6 @@
       "@stacks/blockchain-api-client": ["client/src"]
     }
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "git-info-generator.mjs"],
```

Comment:
What's the reason for this change?

Reply:
oh, that's slipped in, I don't think we need that anymore.